src/utils/helper-functions/utils.ts

```diff
@@ -1,5 +1,6 @@
 import type { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils'
 import {
+  isCallExpression,
```

Comment:
@timdeschryver While doing this, I noticed that many of the guards we implemented here already exist in [**eslint-etc**](https://github.com/cartant/eslint-etc/blob/ee59baa6caf2ddcfd20cf398acabe8d75df4a41a/source/is.ts). If you agree, I can send a follow-up PR replacing these.

Reply:
I was waiting on the TS eslint package to create these guards.
They already have some at https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/experimental-utils/src/ast-utils/predicates.ts but not all of them :( 
Let's create a new issue for this and evaluate later what we want to do with it?

Reply:
Perfect. I'll raise an issue so we don't forget this.